icssc-projects/peterportal-client
# Added redux toolkit and fixed review adding issue

Added groundwork for using redux.

## Description
Setup global store to wrap around app. Created a Review Slice to dynamically add a review. 

## Steps to verify/test this change:
1. Go to a Course Page
2. Add a Review
3. Check that review is added without having to reload the page
